.drone.yml

```diff
@@ -8,6 +8,7 @@ trigger:
     include:
       - 'refs/tags/v*'
       - 'refs/heads/main'
+      - 'refs/heads/DOP-3779'
```

Comment:
Small obligatory reminder to remove before merging :P

Reply:
done!